roles.json

```
@@ -267,6 +267,25 @@
             ]
         }
     },
+    {
+        "role": "DevOps and Operations",
```

```suggestion
        "role": "DevOps and Issue Triage Maintainer",
```
I'm not a fan of "DevOps and Operations" because it's operations twice.  But maybe this captures it better?  Not that I'm thinking "issue triage maintainer" on the theory that this is not necessarily the people who do all the triaging (even though in practice right now, it's definitely the lion's share!) but potentially it can be the people who oversee a larger group doing triaging.

---

I think this is perhaps too specific.

---

In reality it is "repo janitor" but it doesn't sound nice. 😉

---

Indeed, this is the role that goes into the gutter of things :)

issue triage is something different though, that should very much include the contextual package maintainers and that is need to be done regularly. devops people cannot really go into the issues and triage them besides an initial classification (and frankly that part is the easiest entry-level thing that can be outsourced to brand new people to I would not tie it with admin access). 

tbh I quite like "devops and operations" as that is something the industry knows and understand, coming up a flowery language to describe it on the other hand would not have the same effect.

---

Ok I see these points.  My main reason for adding "issue triage maintainer" is because the roles specifically say "Perform triaging of issues and pull requests" and that is *not* normally a DevOps role.  But DevOps is not well definied in s/w engineering IMHO anyway so :shrug:.

But as a compromise, what about just
```suggestion
        "role": "DevOps Maintainer",
```
 then? I still think "DevOps and Operations" is like "SDSS Survey".

(And even if you don't like that, I think some other re-wording is necessary for it to be a *person in a role* rather than a role.  That is, to be consistent with the other roles it should be "DevOps maintainer" "DevOps specialist", "DevOps Ninja" or similar, not just "DevOps".)

---

it's not really maintainer, as part of the role is to get things fixed up, worked around, etc, not just fixing up the tools that are needed for things to get fixed up. Thus my liking for devops and operations, even though it's RAS (but if it works for sdss, it could work for this, too :) )
While it somewhat sounds weird, could live with the "devops specialist", too, but I feel "ninja" is very much curturaly inappropriate.

---

Ok, I'm cool with either
```suggestion
        "role": "DevOps Specialist",
```
or 
```suggestion
        "role": "DevOps and Operations Specialist",
```
then.  I'm open to really any "person" ending there too.